docs/en/wallet-solution.rst

```diff
@@ -92,7 +92,7 @@ Header
     * - **Key**
       - **Value**
     * - alg
-      - Algorithm used to verify the token signature. It MUST be one of the possibile values indicated `here <https://italia.github.io/eudi-wallet-it-docs/versione-corrente/en/algorithms.html>`_ (e.g., ES256).
+      - Algorithm used to verify the token signature. It MUST be one of the possibile values indicated here [here](https://italia.github.io/eudi-wallet-it-docs/versione-corrente/en/algorithms.html) (e.g., ES256).
```

Comment:
this was the RST href while the one your proposing here is the Markdown href type